src/etl/sars-cov-2/steps/add-vaccination-data-to-estimate-step.ts

```diff
@@ -1,10 +1,100 @@
 import { MongoClient } from "mongodb";
+import { add, sub } from 'date-fns';
```

Comment:
If you're wondering why I didn't also do this for the positive case data, that data is far more complete and has fewer holes. The vaccination data is full of holes.